crucible-llvm/src/Lang/Crucible/LLVM/Translation/Constant.hs

```diff
@@ -805,7 +805,9 @@ evalConv expr op mt x = case op of
       , DoubleConst d <- x
       -> return $ IntConst w (BV.mkBV w (truncate d))
 
-    L.UiToFp
+    -- NB: The `nneg` is ignored, as this flag is never enabled in constant
+    -- expressions, only in instructions.
```

Comment:
Should we perhaps `assert` that the flag is not enabled? (Here and elsewhere)

Reply:
Yes, good call. I've converted these to `assert`ions.